internal/pkg/manifest/validate.go

```diff
@@ -44,9 +44,132 @@ func (l *LoadBalancedWebServiceConfig) Validate() error {
 	if err = l.PublishConfig.Validate(); err != nil {
 		return fmt.Errorf(`validate "publish": %w`, err)
 	}
-	for i := 0; i < len(l.TaskDefOverrides); i++ {
-		if err = l.TaskDefOverrides[i].Validate(); err != nil {
-			return fmt.Errorf(`validate taskdef_overrides[%d]: %w`, i, err)
+	for ind, taskDefOverride := range l.TaskDefOverrides {
+		if err = taskDefOverride.Validate(); err != nil {
+			return fmt.Errorf(`validate taskdef_overrides[%d]: %w`, ind, err)
+		}
+	}
+	return nil
+}
+
+// Validate returns if BackendServiceConfig is configured correctly.
+func (b *BackendServiceConfig) Validate() error {
+	var err error
+	if err = b.ImageConfig.Validate(); err != nil {
+		return fmt.Errorf(`validate "image": %w`, err)
+	}
+	if err = b.ImageOverride.Validate(); err != nil {
```

Comment:
Just curious why some errors are wrapped and others (here and below) aren't.

Reply:
It depends on if the struct is `inlined` or not (for better error msg helping them to debug)